doc/whats_new/v1.6.rst

```diff
@@ -182,6 +182,9 @@ Changelog
   estimator without re-fitting it.
   :pr:`29067` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Fix| Improve error message when :func:`model_selection.RepeatedStratifiedKFold.split` is called without a `y` argument
```

Comment:
I have checked the [rendered doc](https://output.circle-artifacts.com/output/job/eb54c496-9a50-4eaa-bf32-89ddf0cb30a6/artifacts/0/doc/whats_new/v1.6.html#sklearn-model-selection) ("check the rendered doc" CI status and then navigate to the file you care about) and it looks that you need to add a dot:
```suggestion
- |Fix| Improve error message when :func:`model_selection.RepeatedStratifiedKFold.split` is called without a `y` argument.
```